.github/ISSUE_TEMPLATE/3_support_question.md

```diff
@@ -1,11 +1,22 @@
 ---
 name: ⛔ I need help
-about: See https://www.prestashop.com/support for questions about using PrestaShop
+about: For questions about using PrestaShop
 
 ---
 
 We use GitHub issues only to discuss about bugs in the PrestaShop project and new features.
-If you have questions about using PrestaShop or third-party modules, or if you need help
-with your shop, please refer to one of our support plans: https://www.prestashop.com/support
+
+If you have questions about using PrestaShop, you might find the answer here:
+- User's guide - http://doc.prestashop.com/display/PS17/User+Guide
+- Developer documentation - https://devdocs.prestashop.com/
+
+If you need help with your shop or third-party modules, please consider one of our support plans: https://www.prestashop.com/support
```

Comment:
I have problem with our : who is `we` ? The Open Source Project ? The PrestaShop Company ? The Maintainers Team ?

Reply:
PS company

Reply:
In this case, the `our` must be contextualized

Reply:
`$this->context->we` ?

Reply:
:trollface: 

```suggestion
If you need help with your shop or third-party modules, please consider the PrestaShop Company support plans: https://www.prestashop.com/support
```